src/documents/activities/fr/Create-a-Thon.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Le Créathon
+eleventyNavigation:
+    parent: Activities
+    key: Create-a-Thon
+---
+
+Un créathon rassemble un groupe varié de personnes dans le but de contribuer collectivement à la conception d'un
+système, produit ou service. L'essentiel est d'impliquer un groupe de participants variés, incluant des personnes dont
+les besoins attendent une solution (et qui trouvent qu'utiliser un produit ou service est difficile voire impossible),
+ainsi que des personnes dont les points de vue ne sont généralement pas pris en compte car elles ne correspondent pas
+aux modèles « typiques », « moyens » ou au « public visé » prévu.
+
+{% figure "/assets/images/CaT-01.jpg", "La photographie de quatre personnes qui participent à un créathon avec des jeux
+de construction." %}
+
+*Illustration 1 :* Un groupe collabore à la création d'un circuit de billes en partant de règles simples.
+
+{% endfigure %}
+
+Le Créathon encourage les participants à créer un objet en « concevant par métaphore » et à l'utiliser comme moyen de
+transposer leurs idées dans le système, produit ou service qui est en train d'être conçu ou testé. Ça permet de limiter
+l'objet physique et de laisser les participants trouver des liens qui n'auraient pas été aussi clairement établis si on
+leur avait directement demandé de concevoir une solution. Ça permet également aux concepteurs dont ce n'est pas le
+domaine d'expertise de s'investir et d'apporter plus facilement leur contribution sans nécessiter un bagage de
+connaissances préalables.
+
+Durant cet événement collaboratif, les participants devraient en permanence se décrire le processus entre eux,
+s'entraider quand les compétences sont différentes et expliquer leur façon de faire. Devoir composer avec les besoins
+de chacun suscite des situations et des idées précieuses qu'il faut noter, rassembler et mettre en pratique pour
+améliorer la conception du système, produit ou service sur lequel on travaille. C'est important d'éviter de grouper les
+points de vue et expériences dans une série de moyennes, ou de limiter les différences pour favoriser les similitudes.
+Chaque histoire a son importance et contribue à enrichir les concepts et les idées.
+
+Il est possible, et même souhaitable, d'organiser plusieurs Créathons à différentes étapes du processus de conception
+pour produire de nouvelles idées.
+
+{% figure "/assets/images/CaT-02.jpg", "La photographie d'une personne qui présente le travail de son groupe à
+l'occasion d'un créathon." %}
+
+*Illustration 2 :* Un participant présente le concept de son groupe qui avait pour tâche de créer une version
+informatique du circuit de billes.
```

Comment:
When rendered, a `<br/>` is injected between the lines causing a break.

![image](https://user-images.githubusercontent.com/564762/135520367-f2ab88b0-436c-4789-94b5-350bb942c541.png)

Reply:
This is caused by https://github.com/fluid-project/eleventy-plugin-fluid/issues/76